packages/panda-san/src/variables/container.scss

```diff
@@ -1,6 +1,6 @@
 $container-gutters: (
-  widescreen: 32px,
```

Comment:
I believe when they are reordered like this the breakpoints do not correctly apply due to the order they're declared in. Is `container` still applying correct gutter width for you with this change?

Reply:
Did you verify this?